src/syntax/index.ts

```diff
@@ -0,0 +1,38 @@
+import type { Root } from "@wooorm/starry-night";
+import { createStarryNight, common } from "@wooorm/starry-night";
+import sourceSvelte from "@wooorm/starry-night/lang/source.svelte.js";
+import sourceSolidity from "@wooorm/starry-night/lang/source.solidity.js";
+import sourceToml from "@wooorm/starry-night/lang/source.toml";
+import sourceErlang from "@wooorm/starry-night/lang/source.erlang.js";
+import sourceDockerfile from "@wooorm/starry-night/lang/source.dockerfile";
+import sourceAsciiDoc from "@wooorm/starry-night/lang/text.html.asciidoc";
+
+export type MaybeHighlighted = Root | undefined;
+type StarryNight = Awaited<ReturnType<typeof createStarryNight>>;
+
+// This takes 33 as common defined languages defined in node_modules/@wooorm/starry-night/lib/common.js
+// NOTICE: Currently we always load these 33 languages, eventually we can/should load them on demand.
```

Comment:
I think it's pretty obvious what's happening here, the comment is redundant.

Reply:
84ace38822a5594d6c793200e5af2e81b79154da